UI/src/view/point_asset/trafficSignForm.js

```diff
@@ -626,7 +627,11 @@
 
     var singleChoiceForPanels = function (property) {
       var publicId = _.head(property);
-      var propertyValue = (!_.isEmpty(_.last(property))) ? '' : _.last(property);
+      var propertyValue;
+      if (_.last(property) === 0)
```

Comment:
Maybe switch to

if ( _.isEmpty( _.last(property) )

Reply:
lodash's isEmpty does not support numbers, so for example isEmpty(1) will return true